contrib/clojure-package/README.md

```diff
@@ -0,0 +1,197 @@
+# Clojure MXNet
+
+A clojure package to the MXNet Deep Learning library
+
+## Introduction
+
+MXNet is a first class, modern deep learning library that AWS has officially picked as its chosen library. It supports multiple languages on a first class basis and is incubating as an Apache project.
+
+The motivation for creating a Clojure package is to be able to open the deep learning library to the Clojure ecosystem and build bridges for future development and innovation for the community. It provides all the needed tools including low level and high level apis, dynamic graphs, and things like GAN and natural language support.
+
+For high leverage, the Clojure package has been built on the existing Scala package using interop. This has allowed rapid development and close parity with the Scala functionality. This also leaves the door open to directly developing code against the jni-bindings with Clojure in the future in an incremental fashion, using the test suites as a refactoring guide.
+
+## Current State and Plans
+
+The Clojure package is nearing the end of its first development milestone which is to achieve a close parity with the Scala package and to potentially be included into the main project for official Clojure language support.
+
+What is needed now is alpha testing on both OSX and Linux to discover any bugs, rough edges, and generally harden it before an official PR is opened on the main project.
+
+Help with this effort is greatly appreciated and contributors will be recognized in the project README.
+
+Testing instructions can be found in the Testing.md
+
+## Getting Started
+
+The following systems are supported:
+
+- OSX cpu
+- Linux cpu
+- Linux gpu
+
+There are two ways of getting going. The first way is the easiest and that is to use the pre-built jars from Maven. The second way is to build from source. In both cases, you will need to load the prereqs and dependencies, (like opencv).
+
+It's been tested on AWS Deep Learning AMI and OSX High Sierra 10.13.4
+
+
+### Prerequisites
+
+**If you are using the AWS Deep Learning Ubuntu or Linux AMI you should be good to go without doing anything on this step.**
+
+
+Follow the instructions from https://mxnet.incubator.apache.org/install/osx_setup.html or https://mxnet.incubator.apache.org/install/ubuntu_setup.html
+about _Prepare Environment for GPU Installation_
+and _Install MXNet dependencies_
+
+The only difference is that for OSX you will need to install opencv2 to use the prebuilt jar
+`brew search opencv@2`
+`brew install opencv@2`
+
+
+### Use Prebuilt Jars
+There are deployed jars on Clojars for each supported system
+
+* `[org.apache.clojure-mxnet/clojure-mxnet-linux-gpu "0.1.1-SNAPSHOT"]`
```

Comment:
Who is building these packages?   How is the version number related to the mxnet version?

Can we add instructions to build the package as well? Something like when you build a scala package in the local ivy repo.

Reply:
That's a good point. The section was taken from the README in the external git repo that I had for testing. I had built and deployed those jars manually and pushed them to clojars.

I'm not sure how the build will happen within the project when it gets merged long term.  We will need some discussion on that. For the short term, I can continue to build the jars manually and push them to clojars for general users to consume and give feedback on. 

I definitely can document the process that I use to do the build and deploy 👍